pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -155,6 +155,10 @@ func NetlinkFlowToAntreaConnection(conn *conntrack.Flow) *flowexporter.Connectio
 		SourcePodName:           "",
```

Comment:
This file probably worth a separate pkg under pkg/agent/util/conntrack. Again not saying we must do in this PR.